plugins.go

```diff
@@ -0,0 +1,21 @@
+package connect
+
+import "net/http"
+
+// Plugin represents a Kafka Connect connector plugin
+type Plugin struct {
+	Class   string `json:"class"`
+	Type    string `json:"type"`
+	Version string `json:"version"`
+}
+
+// ListPlugins retrieves a list of the installed plugins.
+//
+// See: https://docs.confluent.io/current/connect/references/restapi.html#get--connector-plugins-
+func (c *Client) ListPlugins() ([]*Plugin, *http.Response, error) {
```

Comment:
I would prefer to return `[]Plugin` here rather than pointers, consistent with `GetConnectorTasks`. Since these are small structs holding only simple types and the collection will be small, sticking to the stack should be fine.

Also, I think I'd go with `GetPlugins()` as the method name—trying to recall why there is the seeming inconsistency of e.g. `ListConnectors()` vs. `GetConnectorTasks`, I believe it was because the `GET /connectors` API returns an array of scalar names, it's not a very "resourceful" REST endpoint and the method thus doesn't pair so well with `GetConnector(name string)`. For entity-like resource collections I prefer the `GetResources` naming convention.